client/lib/user/user.js

```diff
@@ -67,31 +68,42 @@ User.prototype.initialize = function() {
 		// Store the current user in localStorage so that we can use it to determine
 		// if the logged in user has changed when initializing in the future
 		if ( this.data ) {
-			this.clearStoreIfChanged( this.data.ID );
-			store.set( 'wpcom_user', this.data );
+			if ( ! this.data.abtests ) {
```

Comment:
Should this part be in this PR? seems like it's only for the bootstrapped user load

Reply:
With the latest change loading variations from `/me`, this should no longer be an issue.